resources/assets/js/components/UserEditForm.vue

```diff
@@ -27,29 +27,29 @@
         <h3>Additional Information</h3>
 
         <div class="form-group row">
-          <label for="user-preferredname" class="col-sm-2 col-form-label">Preferred First Name</label>
+          <label for="user-preferredname" class="col-sm-2 col-form-label">Preferred Name</label>
```

Comment:
These changes clobber a PR I merged recently. This field should be `preferred_first_name`.